src/crystalla/matrix.cr

```diff
@@ -216,6 +241,27 @@ module Crystalla
       Matrix.rows rows
     end
 
+    class SVD
```

Comment:
I think it's not used, maybe it was used in a previous version but now a tuple is returned. Maybe SVD was ok because the members have names (I'd use `record SVD u, s, vt`).